service/src/integration-test/resources/fhir/encounter_valid.json

```diff
@@ -0,0 +1,22 @@
+{
+    "resourceType": "Bundle",
+    "type": "transaction",
+    "entry": [
+        {
+            "fullUrl": "urn:uuid:14200fa9-00f9-4106-8ce2-11a09c446a93",
+            "resource": {
+                "resourceType": "Encounter",
+                "status": "finished",
+                "serviceProvider": {
+                    "reference": "urn:uuid:23b2c1e7-ebb4-48a0-8e18-1e2716b85752"
+                }
```

Comment:
Are we supposed to amend this json when adding the mapper bricks to our app, so new elements to our bundle? Do I git it right?

Reply:
Until we implement reading from queue - yes. This way we can be sure that we produce a valid FHIR message. Once reading from queue is implemented it'll just work automatically.